euphonic/io.py

```diff
@@ -26,12 +26,14 @@ def _to_json_dict(dictionary: Dict[str, Any]) -> Dict[str, Any]:
 
 
 def _from_json_dict(dictionary: Dict[str, Any],
-                    type_dict: Dict[str, Type[Any]] = {}) -> Dict[str, Any]:
+                    type_dict: Dict[str, Type[Any]]|None = None) -> Dict[str, Any]:
```

Comment:
Most linters would suggest:
```suggestion
                    type_dict: Dict[str, Type[Any]] | None = None) -> Dict[str, Any]:
```
 Also, given you're using `|` unions, do the types need to be title-case?

Reply:
Coming back to needless typing imports later 😄